lemminx-maven/pom.xml

```diff
@@ -188,6 +188,16 @@
 				<enabled>true</enabled>
 			</releases>
 		</repository>
+		<repository>

```

Comment:
Please remove it as we use the release now.

Reply:
ok